backends/aerospike.go

```diff
@@ -13,59 +15,111 @@ import (
 const setName = "uuid"
 const binValue = "value"
 
+type AerospikeDBClient interface {
+	NewUuidKey(namespace string, key string) (*as.Key, error)
+	Get(key *as.Key) (*as.Record, error)
+	Put(key *as.Key, value string, ttlSeconds int) error
+}
+
 type Aerospike struct {
+	client *as.Client
```

Comment:
What's the rationale behind extracting the Aerospike Client into it's own struct?

Reply:
In order to mock the `AerospikeDBClient interface` in `aerospike_test.go` I realize this seems repetitive, do you suggest an alternative?

Reply:
Please see my comment above the `AerospikeDB` interface. I am not sure why we need that and so based on that we can see if we need this anymore